### ISSUE TITLE
following steps in README results in certificate errors

### ISSUE THREAD

**Issue by lbergelson** [github/lfs-test-server#25]
I tried following the README and creating a certificate to use for https access.  Following the steps in the README results in an error. 

```
LocalWorkingDir=/Users/louisb/tmp/test-gitlfs
LocalGitDir=/Users/louisb/tmp/test-gitlfs/.git
LocalMediaDir=/Users/louisb/tmp/test-gitlfs/.git/lfs/objects
TempDir=/Users/louisb/tmp/test-gitlfs/.git/lfs/tmp
GIT_PREFIX=

Error for POST https://localhost:9999/user/repo/objects

Post https://localhost:9999/user/repo/objects: x509: certificate signed by unknown authority
goroutine 16 [running]:
github.com/github/git-lfs/lfs.Stack(0x0, 0x0, 0x0)
	/Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/lfs/errors.go:85 +0x89
github.com/github/git-lfs/commands.logPanic(0x6674d0, 0xc2082dc360, 0x69da00, 0x0, 0x0)
	/Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/commands.go:156 +0xf17
github.com/github/git-lfs/commands.handlePanic(0x6674d0, 0xc2082dc360, 0x0, 0x0)
	/Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/commands.go:108 +0x6c
github.com/github/git-lfs/commands.LoggedError(0x6674d0, 0xc2082dc360, 0xc2082d4000, 0x37, 0x0, 0x0, 0x0)
	/Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/commands.go:65 +0x89
github.com/github/git-lfs/commands.prePushCommand(0x594800, 0xc208044640, 0x2, 0x2)
	/Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/commands_pre_push.go:97 +0x63f
github.com/spf13/cobra.(*Command).execute(0x594800, 0xc208044620, 0x2, 0x2, 0x0, 0x0)
	/Users/rick/github/git-lfs/.vendor/src/github.com/spf13/cobra/command.go:355 +0x17c
github.com/spf13/cobra.(*Command).findAndExecute(0x594700, 0xc20800e010, 0x3, 0x3, 0x0, 0x0)
	/Users/rick/github/git-lfs/.vendor/src/github.com/spf13/cobra/command.go:334 +0xd3
github.com/spf13/cobra.(*Command).Execute(0x594700, 0x0, 0x0)
	/Users/rick/github/git-lfs/.vendor/src/github.com/spf13/cobra/command.go:408 +0x7b0
github.com/github/git-lfs/commands.Run()
	/Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/commands.go:80 +0x2a
main.main()
	/Users/rick/github/git-lfs/cmd/git-lfs/git-lfs.go:8 +0x1a
```

I assume that having 
```
 [http]
        selfverify = false
```

in the .gitconfig is supposed to avoid this, but it doesn't seem to work.  Switching from `https` to `http` resolved this error.

**Comment by rubyist** [comment]
Ahh, that should be `sslverify = false`, not `selfverify`. I'll fix up the readme, thanks for the report!

**Comment by rubyist** [comment]
Fixed up in #28
